xml/Microsoft.Build.Tasks/Hash.xml

```diff
@@ -14,7 +14,12 @@
   </Base>
   <Interfaces />
   <Docs>
-    <summary> Generates a hash of a given group of items. Metadata is not considered in the hash.</summary>
+    <summary>
+            Generates a hash of a given ItemGroup items. Metadata is not considered in the hash.
+            <remarks>
```

Comment:
```suggestion
```

Remarks shouldn't be nested in summary tags.

Reply:
Here's the preview: https://review.docs.microsoft.com/en-us/dotnet/api/microsoft.build.tasks.hash?view=msbuild-15&branch=pr-en-us-4939.

The remarks should ideally be moved to the Remarks section.